server/routes/createPlan/abilityToWork/index.test.ts

```diff
@@ -1,15 +1,13 @@
 import { Router } from 'express'
 import Controller from './abilityToWorkController'
 import getPrisonerByIdResolver from '../../../middleware/resolvers/getPrisonerByIdResolver'
-import getCiagPlanByIdResolver from '../../../middleware/resolvers/getCiagPlanByIdResolver'
```

Comment:
The next set of changes in this diff are for the route definition files for each page, and their unit tests; where I have removed the use of `getCiagPlanByIdResolver` altogether.

Personally I think these tests are completely pointless and are not really testing anything. I've left them, but will discuss their future in a slack conversation

Reply:
This comment is now outdated - we have subsequently decided to delete all of the unit tests for the route setups, as they serve little purpose or value.